microservices-orders/docs/index.rst

```diff
@@ -177,13 +190,7 @@ Exercise 0: Run end-to-end demo
 
 Running the fully working demo end-to-end provides context for each of the later exercises.
 
-1. Clone the `examples GitHub repository <https://github.com/confluentinc/examples>`__:
-
-.. sourcecode:: bash
-
-   git clone https://github.com/confluentinc/examples
-
-2. Start the demo
+1. Start the demo
```

Comment:
Consider using the `#.` format or numbered lists.

Reply:
I used it where I could